plugins/module_utils/database.py

```diff
@@ -140,3 +142,64 @@ def mysql_quote_identifier(identifier, id_type):
             special_cased_fragments.append(fragment)
 
     return '.'.join(special_cased_fragments)
+
+
+def is_input_dangerous(string):
+    """Check if the passed string is potentially dangerous.
+    Can be used to prevent SQL injections.
+
+    Note: use this function only when you can't use
+      psycopg2's cursor.execute method parametrized
+      (typically with DDL queries).
+    """
+    if not string:
+        return False
+
+    # 1. '"' in string and '--' in string or
+    # "'" in string and '--' in string
+    pattern1 = re.compile(r'(\'|\").*--')
```

Comment:
Since none of these re.compiles use variables, you can move them outside of the function so that they are compiled only once, when the module_utils is imported.  That way, calling `is_input_dangerous()` in a loop will be faster.

Reply:
@abadger good point! Thanks, done